content/news/posts/2024/autoupgrade-v6-release.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Autoupgrade module v6.0.0 is Available (1-Click Upgrade)"
```

Comment:
```suggestion
title: "Autoupgrade Module v6.0.0 is Available (1-Click Upgrade)"
```

Reply:
What's the reason behind this suggestion? "Module" is not a part of the name but the regular extension as in the rest of the article.

Reply:
@coffrep capitalization of the title is controlled via CSS, no need to capitalize it in the text

Reply:
Ok fine thanks for the info @kpodemski - yes in English @Quetzacoalt91 , titles have capitalized first letters in each word ;-)